python/social_bot/goal_task.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
```

Comment:
Did you do "pre-commit install" according to https://github.com/HorizonRobotics/SocialRobot/blob/master/CONTRIBUTING.md#workflow

It should automatically insert a full copy right notice like the following instead of one line.
https://github.com/HorizonRobotics/SocialRobot/blob/1b65fda55aaefcd956ee7f1bd60dcbc8c038b5e1/tools/codestyle/copyright.hook#L11-L25

Reply:
Copyright info updated. I did use pre-commit, but it seems that pre-commit won't update if there was already an one-line copyright added by hand.